source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+
+Method 1: Fresh installation of |CL| into a new VM 
+==================================================
+
+The general process for performing a fresh installation of |CL| into a new VM 
+is as follows (with expanded details below):
+
+* Create a new VM and configure its settings
+* Attach the installer ISO to it
+* Install |CL|
+* Detach the installer ISO
+* Power off the VM
+* Enable EFI boot support
+* Power on the VM
+
+Create and configure a new VM 
+*****************************
+
+#.  Start the `VMware Workstation Player` app.
+#.  On the home screen:
+
+    * Click `Create a New Virtual Machine`.
+
+      |vmware-player-01|
+
+      Figure 1: VMware Workstation 14 Player - Create a new virtual machine
+   
+#.  On the `Welcome to the New Virtual Machine Wizard` step:
+
+    * Select `Installer disc image file (iso)`.
+    * Click `Browse` and select the uncompressed |CL| installer ISO. 
+    * Click `Next`.
+
+      |vmware-player-02|
+
+      Figure 2: VMware Workstation 14 Player - Select |CL| installer ISO   
+
+#.  On the `Select a Guest Operating System` step:
+
+    * Set `Guest operating system` to `Linux`.
+    * Set `Version` to `Other Linux 3.x or later kernel 64-bit`.
+    * Click `Next`.
+
+      |vmware-player-03|
+
+      Figure 3: VMware Workstation 14 Player - Select guest operating system type
+
+#.  On the `Name the Virtual Machine` step:
+
+    * Give it a name.
+    * Click `Next`.
+
+      |vmware-player-04|
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#.  On the `Specify Disk Capacity` step:
+
+    * Set the desired disk size.
+    * Click `Next`.
+
+      |vmware-player-05|
+
+      Figure 5: VMware Workstation 14 Player - Set disk capacity
+
+#.  On the `Ready to Create Virtual Machine` step:
+
+    * Click `Customize Hardware...`
+    
+      |vmware-player-06|
+
+      Figure 6: VMware Workstation 14 Player - Customize hardware
+
+    * Select `Processors` > `Virtualization engine` and 
+      check `Virtualize Intel VT-x/EPT or AMD-V/RVI`.
+
+      |vmware-player-07|
+
+      Figure 7: VMware Workstation 14 Player - Set virtualization engine option
+
+    * Click `Close`.
+
+    * Click `Finish`.
+
+Install |CL| into the new VM
+****************************
+
+#.  Select the newly created VM and click `Play virtual machine`.  
+
+    |vmware-player-08|
+
+    Figure 8: VMware Workstation 14 Player - Power on virtual machine
+
+#.  Follow the :ref:`bare-metal-install` (starting at the `Install Clear Linux 
+    on your target system` section) guide to complete the installation of 
+    |CL|.
+
+#.  After the installation is complete, follow the |CL| instruction to reboot it.  
+    This will restart the installer again.  Follow the next section.  
+
+Reconfigure the VM settings to boot the newly installed |CL|
+************************************************************
+
+#.  Enable the mouse pointer.
+
+    - Press `<CTRL>` + `<ALT>` to enable the mouse pointer.
```

Comment:
Should this be a bullet rather than a "-"? 
Remove "to enable the mouse pointer".